authorize/grpc.go

```diff
@@ -98,7 +85,7 @@ func (a *Authorize) forceSync(ctx context.Context, ss *sessions.State) error {
 	}
 	s := a.forceSyncSession(ctx, ss.ID)
 	if s == nil {
-		return nil
+		return errors.New("session not found")
```

Comment:
This should cause stale sessions (that is, sessions tied to a databroker which was restarted) to be treated as empty sessions. Users will be redirected to the login page.

There's potential here for a redirect loop. (it *should* work...)